.github/workflows/build-windows.yml

```diff
@@ -2,21 +2,29 @@ name: Build Windows
 
 on:
   pull_request:
-    paths-ignore:
-      - README.md
-      - .github/**
-      - '!.github/workflows/build-windows.yml'
+    paths:
+      - lib/**/*.js
+      - Makefile
+      - src/**/*.cc
+      - src/**/*.h
+      - tools/gyp/**
+      - tools/test.py
```

Comment:
Actually, we don't care for `test.py` changes either, since it's only useful when running tests
```suggestion
```